tests/integration/cmd_storage_test.go

```diff
@@ -32,97 +30,6 @@ var _ = Describe("odo storage command tests", func() {
 		})
 	})
 
-	Context("when running storage command without required flag(s)", func() {
```

Comment:
Is there any reason we are removing this test?

Reply:
Tests that have been removed without a comment have a devfile counterpart already present hence can be deleted